stable/traefik/templates/rbac.yaml

```diff
@@ -3,6 +3,10 @@ kind: ServiceAccount
 apiVersion: v1
 metadata:
   name: {{ template "traefik.fullname" . }}
+  {{- with .Values.rbac.serviceAccount.annotations }}
```

Comment:
This should be wrapped in an If, so that this is not mandatory

Reply:
If annotations are empty, the `with` clause will not execute.

Reply:
Done